### PR TITLE
Update plan-um-migration.md

### PR DESCRIPTION
**Pull request by dariomws306**
https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/issues/3281

### CODE REVIEW

**Comment by scanum** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Skype/SfbHybrid/hybrid/plan-um-migration.md [hunk 1]
```diff
@@ -38,7 +38,7 @@ Cloud Voicemail doesn't provide voice messaging functionality to Private Branch
 
 With Skype for Business Server 2019, we're moving from Exchange Online UM to Cloud Voicemail. When a user is moved to a Skype for Business 2019 server, they'll automatically start using Cloud Voicemail when configured for hosted voicemail. If you're currently using Exchange Online UM, you don't need to do anything other than move a user to Skype for Business Server 2019 to start using Cloud Voicemail. However, there are some changes to functionality you need to be aware of:
 
-- Organizational Auto Attendant (the replacement for auto attendant in Exchange Online UM) isn't available at GA but will be available later in 2018.
+- Cloud Auto Attendant (the replacement for auto attendant in Exchange Online UM). For more information, see [What are Cloud auto attendants?](https://docs.microsoft.com/microsoftteams/what-are-phone-system-auto-attendants).
```
Comment (JohanFreelancer9) [inline Skype/SfbHybrid/hybrid/plan-um-migration.md]:
```suggestion
- Cloud Auto Attendant (the replacement for auto attendant in Exchange Online UM). For more information, see [What are Cloud auto attendants?](https://docs.microsoft.com/microsoftteams/what-are-phone-system-auto-attendants)
```

**Comment by scanum** [comment]
@dstrome PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by dstrome** [comment]
I'm not going to accept the change because it would be inconsistent with the rest of the content in the topic. However, I'm creating a work item for myself to re-work the topic given that some features, such as Org AA, are now available. Thanks for bringing this to my attention @dariomws306 !

David.

**Comment by dariomws306** [comment]
Hi @dstrome,

This update was suggested by Thomas Binder from Microsoft. If you want, I can help you replacing all the occurrences in this article.

Thanks!

**Pull request closed without merging**